src/appinsights-props.js

```diff
@@ -0,0 +1,18 @@
+export var dataKey = 'appinsights';
+
+export class AppinsightsPropsCustomAttribute {
+	static inject = [Element];
+
+	constructor(element) {
+		this.element = element;
+	}
+
+	valueChanged(newValue) {
+		if (!(typeof newValue == 'object' && newValue !== null)) {
```

Comment:
Consider `if(typeof newValue !== 'object' || newValue === null)` instead to make it easier to read.